nextjs-app-v13/src/pages/components/checkbox.tsx

```diff
@@ -0,0 +1,21 @@
+import NavigationLayout from "@/layout/navigation";
+import { PieCheckbox } from "@justeattakeaway/pie-webc/react/checkbox.js";
+import { useState } from "react";
+
+export default function Checkbox() {
+    const [isCheckboxChecked, setIsCheckboxCheck] = useState(false)
+
+    const handleCheckboxChange = () => {
+    setIsCheckboxCheck(current => !current);
```

Comment:
NIT: can we fix the indentation here please? Perhaps we need to add some linting to the app if we don't already have it 😄